src/audio/aria/aria.c

```diff
@@ -162,8 +162,10 @@ static void aria_set_stream_params(struct comp_buffer *buffer,
 	const struct ipc4_audio_format *audio_fmt = &mod->priv.cfg.base_cfg.audio_fmt;
 
 	ipc4_update_buffer_format(buffer, audio_fmt);
-#ifndef ARIA_GENERIC
+#ifdef ARIA_HIFI3
 	audio_stream_set_align(8, 1, &buffer->stream);
+#elif ARIA_HIFI5
```

Comment:
Same question as in https://github.com/thesofproject/sof/pull/8682/files#r1469048792 : where and how are these defined?

Reply:
the ARIA_HIFI5 is defined in aria.h (https://github.com/thesofproject/sof/pull/8692) while adding HiFi5 implementation.

Reply:
I was confused because 307921c7ac24 was using the new way but its parent was still using the old way. Resolved, everything new now.